rust-runtime/aws-smithy-http/Cargo.toml

```diff
@@ -29,16 +32,30 @@ hyper = "0.14"
 # ByteStream internals
 futures-core = "0.3.14"
 tokio = { version = "1.6", optional = true }
-tokio-util = { version = "0.6", optional = true}
+tokio-util = { version = "0.6", optional = true }
+
+# Checksum algos
+crc32fast = "1.3"
+crc32c = "0.6"
+sha1 = "0.10"
+sha2 = "0.10"
```

Comment:
I think ideally we'd keep this separate so we didn't compile all these deps when they weren't needed